js/flightlog_fields_presenter.js

```diff
@@ -26,6 +26,16 @@ function FlightLogFieldPresenter() {
         'axisF[1]': 'PID Feedforward [pitch]',
         'axisF[2]': 'PID Feedforward [yaw]',
 
+        'axisB[all]': 'PID Boost',
+        'axisB[0]': 'PID Boost [roll]',
+        'axisB[1]': 'PID Boost [pitch]',
+        'axisB[2]': 'PID Boost [yaw]',
+
+        'axisO[all]': 'PID Offset',
+        'axisO[0]': 'PID Offset [roll]',
+        'axisO[1]': 'PID Offset [pitch]',
+        'axisO[2]': 'PID Offset [yaw]',
```

Comment:
Offset is only for Roll and Pitch.

Reply:
Yes, I know that, but firmware exposes this for yaw axis anyway.
- https://github.com/rotorflight/rotorflight-firmware/blob/master/src/main/blackbox/blackbox.c#L217
- https://github.com/rotorflight/rotorflight-firmware/blob/master/src/main/blackbox/blackbox.c#L741
- https://github.com/rotorflight/rotorflight-firmware/blob/master/src/main/blackbox/blackbox.c#L932
If it will be removed there, then it will not appear in viewer.

Reply:
Fair enough, let's keep it then